README.md

```diff
@@ -2,6 +2,39 @@
 
 [![Scala.js](https://www.scala-js.org/assets/badges/scalajs-0.6.6.svg)](https://www.scala-js.org/)
 
+## Installing
+
+First clone this repository locally and then publish locally
+```
+> ++SCALA_VERSION
+> seleniumJSEnv/publishLocal
+```
+
+## Usage
+Simply add the following lines to your sbt settings:
+```
+libraryDependencies += "org.scala-js" %%% "scalajs-env-selenium" % "0.0.1"
```

Comment:
`-SNAPSHOT`

Also, this line should go in `project/plugins.sbt` and use `%%`, not `%%%`.